src/main/java/uk/gov/companieshouse/acsp/models/dao/AMLSupervisoryBodiesDao.java

```diff
@@ -1,35 +1,37 @@
 package uk.gov.companieshouse.acsp.models.dao;
 
-public enum AMLSupervisoryBodiesDao {
-    ACCA("Association of Chartered Certified Accountants (ACCA)"),
-    AAT("Association of Accounting Technicians (AAT)"),
-    AIA("Association of International Accountants (AIA)"),
-    ATT("Association of Taxation Technicians (ATT)"),
-    CILEX("Chartered Institute of Legal Executives (CILEx)"),
-    CIMA("Chartered Institute of Management Accountants (CIMA)"),
-    CIOT("Chartered Institute of Taxation (CIOT)"),
-    CLC("Council for Licensed Council for Licensed (CLC)"),
-    DENI("Department for the Economy Northern Ireland"),
-    FA("Faculty of Advocates"),
-    FOAC("Faculty Office of the Archbishop of Canterbury"),
-    FCA("Financial Conduct Authority "),
-    GC("Gambling Commission"),
-    GCB("General Council of the Bar"),
-    GCBNI("General Council of the Bar of Northern Ireland"),
-    IAB("Institute of Accountants and Bookkeepers (IAB)"),
-    IPA("Insolvency Practitioners Association (IPA)"),
-    ICB("Institute of Certified Bookkeepers"),
-    ICAEW("Institute of Chartered Accountants in England and Wales (ICAEW)"),
-    ICAI("Institute of Chartered Accountants in Ireland (ICAI)"),
-    ICAS("Institute of Chartered Accountants of Scotland (ICAS)"),
-    IFA("Institute of Financial Accountants (IFA)"),
-    LCNI("Law Society of Northern Ireland"),
-    LSS("Law Society of Scotland"),
-    SRASEW("Solicitors Regulation Authority (SRA) Solicitors in England and Wales"),
-    HMRC("Her Majesty's Revenue and Customs");
-
-    public final String label;
-    private AMLSupervisoryBodiesDao(String label) {
-        this.label = label;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.acsp.models.enums.AMLSupervisoryBodies;
+
+public class AMLSupervisoryBodiesDao {
+    @JsonProperty("aml_supervisory_body")
+//    private AMLSupervisoryBodies amlSupervisoryBody;   //this needs to be fixed in web
+    private String amlSupervisoryBody;
```

Comment:
Is this not of type Enum anymore?

Reply:
From frontend, we are getting data like "Association of Chartered Certified Accountants (ACCA)", though we are looping on enums to display the screen.
Backend is expecting ACCA.

I think the front end needs to be fixed to give back ACCA. This might need changes to the view or the model which have enums. Better fix it in a separate ticket.